astropy/time/setup_package.py

```diff
@@ -3,9 +3,9 @@
 # Copied from astropy/convolution/setup_package.py
 
 import os
-from setuptools import Extension
```

Comment:
Same comment as https://github.com/astropy/astropy/pull/13699#pullrequestreview-1112310333 😅

Reply:
Yes. Darn.

Reply:
Either this PR waits for #13699, or we merge now and let isort fix here and there at some later merge. The next time someone pushes the result of ``isort .`` this will be cleaned up.

Reply:
Let's defer this to https://github.com/astropy/astropy/issues/13718 . Thanks!